js/script.js

```diff
@@ -85,9 +85,18 @@ new L.Control.GeoSearch({
 // Here's the Tabletop feed
 // First we'll initialize Tabletop with our spreadsheet
 var jqueryNoConflict = jQuery;
+
 jqueryNoConflict(document).ready(function() {
-  var root_spreadsheet = $("script[src='js/script.js']").data("spreadsheet");
-  initializeTabletopObject(root_spreadsheet);
+  var scriptTag = $("script[src='js/script.js']");
+  var root_spreadsheet = scriptTag.data("spreadsheet");
+
+  fetchActionKitData(args.actionkit, function(actionKitLayerGroups) {

```

Comment:
Made the change to look at `data-map-args` instead of `data-actionkit`. Also removed campaigns from `index.html`, but I tested it first and it works fine.

@ejucovy